src/transformState.js

```diff
@@ -62,7 +62,23 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const data of transforms) {
```

Comment:
transforms - plural
transform - singular

```suggestion
  for (const transform of transforms) {
```
data дуже абстрактна назва

Reply:
Да, сорі, спочатку не думав що мої шедеври буде хтось перевіряти)) Буду робити по нормальному) Виправив